AnnoDesigner/Localization/TranslateExtension.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Windows.Data;
+
+namespace AnnoDesigner.Localization
+{
+    public class TranslateKeyConverter : IValueConverter
+    {
```

Comment:
This is still named Translate, even though the final binding that uses it is called Localize - names should match.